Docs: update docs to reflect new sharing UI (#90630)
grafana/grafana: docs/sources/setup-grafana/image-rendering/_index.md

A background job runs every 10 minutes and removes temporary images. You can configure how long an image should be stored before being removed by configuring the [temp_data_lifetime]({{< relref "../configure-grafana#temp_data_lifetime" >}}) setting.

> You can also render a PNG by hovering over the panel to display the actions menu in the top-right corner, and then clicking **Share > Share link**. When you share an internal panel link, you also have the option to share it as an image.

Reviewer: The sentence starts talking about rendering a PNG. Does it make sense to say 
`When you share an internal panel link, you also have the option to share it as an image.`? 
Should it be something like `and then clicking **Share > Share link**. Render image .`

Author: I think I didn't phrase it well. Let me try again.
```suggestion
You can also render a PNG by hovering over the panel to display the actions menu in the top-right corner, and then clicking **Share > Share link**. The **Render image** option is displayed in the link settings.
```

Reviewer: sounds perfect!
